doc/services/storage/zms/zms.rst

```diff
@@ -414,6 +415,41 @@ verified to make sure that the application could work with one subsystem or the
 both solutions could be implemented, the best choice should be based on the calculations of the
 life expectancy of the device described in this section: :ref:`wear-leveling`.
 
+Recommendations to increase performance
+***************************************
+
+Sector size and count
+=====================
+
+- The total size of the storage partition should be well dimensioned to achieve the best
+  performance for ZMS.
+  All the information regarding the effectively available free space in ZMS can be found
+  in the documentation. See :ref:`free-space`.
+  We recommend choosing a storage partition that can hold double the size of the key-value pairs
+  that will be written in the storage.
+- The size of a sector needs to be dimensioned to hold the maximum data length that will be stored.
+  Increasing the size of a sector will slow down the garbage collection operation which will
+  occur less frequently.
+  Decreasing its size, in the opposite, will make the garbage collection operation faster
+  which will occur more frequently.
+- For some subsystems like :ref:`Settings <settings_api>`, all path-value pairs are split into two ZMS entries (ATEs).
```

Comment:
Why `some subsystems like`? Isn't it only about settings?

Reply:
ZMS could be used directly by any subsystem (bluetooth, secure storage, ...)
This is an example of one subsystem which is Settings